src/Command/Docker/Build.php

```diff
@@ -170,100 +95,31 @@ protected function configure()
                 self::OPTION_MODE,
                 'm',
                 InputOption::VALUE_REQUIRED,
-                sprintf(
-                    'Mode of environment (%s)',
-                    implode(
-                        ', ',
-                        [
-                            ComposeFactory::COMPOSE_DEVELOPER,
-                            ComposeFactory::COMPOSE_PRODUCTION,
-                            ComposeFactory::COMPOSE_FUNCTIONAL,
-                        ]
-                    )
-                ),
-                ComposeFactory::COMPOSE_PRODUCTION
+                sprintf('Mode of environment')
```

Comment:
Looks like using `sprintf` is redundant in this place

Reply:
Fixed